### PR TITLE
Homebrew 3.0.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Add release notes blog post.

Note to self: **check the date is correct before merging these**!

### CODE REVIEW

File: _posts/2021-02-05-homebrew-3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 3.0.0
+author: MikeMcQuaid
+redirect_from: /blog/3.0.0/
+---
+Today I'd like to announce Homebrew 3.0.0. The most significant changes since 2.6.0 are TODO.
```
Comment (SMillerDev) [inline _posts/2021-02-05-homebrew-3.0.0.md:6]: TODO?

File: _posts/2021-02-05-homebrew-3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [`brew untap` of an official tap you don't use (e.g. Homebrew/homebrew-cask) will ensure it's no longer automatically retapped](https://github.com/Homebrew/brew/pull/10366)
+- [`brew casks` is a new command implemented in Bash to speedily output all casks available to install (like `brew formulae`)](https://github.com/Homebrew/brew/pull/10259)
+- [`brew info --cask --json=v2` includes whether a cask is outdated and the currently installed versions](https://github.com/Homebrew/brew/pull/10278)
+- [Fixed a bug where `brew update` could be run every time](https://github.com/Homebrew/brew/pull/10388)
+- [`brew --prefix --installed` is a new flag to `brew --prefix` that will fail if the requested formula is not installed](https://github.com/Homebrew/brew/pull/10266)
+- [We now use an unversioned SDK path on Big Sur to avoid breakage on minor SDK version changes](https://github.com/Homebrew/brew/pull/10072)
+
+Finally:
+
+- [Discourse](https://github.com/Homebrew/brew/pull/9144) was made read-only on January 1st 2021 in favour of [GitHub Discussions](https://github.com/Homebrew/brew/pull/8802).
```
Comment (SMillerDev) [inline _posts/2021-02-05-homebrew-3.0.0.md:31]: Shouldn't these link to the respective forums?

File: _posts/2021-02-05-homebrew-3.0.0.md (hunk 1 above)
Comment (MikeMcQuaid) [reply _posts/2021-02-05-homebrew-3.0.0.md]: correct 😅. fixed in latest push.

File: _posts/2021-02-05-homebrew-3.0.0.md (hunk 2 above)
Comment (MikeMcQuaid) [reply _posts/2021-02-05-homebrew-3.0.0.md]: fixed in latest push.

**Review by SMillerDev — APPROVED** [review]

**Review by dtrodrigues — APPROVED** [review]

**Pull request merged**